core/web/resolver/testdata/config-full-effective.toml

```diff
@@ -0,0 +1,378 @@
+ExplorerURL = 'http://explorer.url'
+InsecureFastScrypt = true
```

Comment:
Any plans to move this one?

Reply:
Yes, tracked in the other PR. I'd say it should be safe for tests to run dev builds? :man_shrugging: